src/test/java/edu/ucsb/courses/documents/statistics/QuarterDeptTests.java

```diff
@@ -0,0 +1,75 @@
+package edu.ucsb.courses.documents.statistics;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+import net.codebox.javabeantester.JavaBeanTester;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class QuarterDeptTests {
+
+    private static Logger logger = LoggerFactory.getLogger(QuarterDeptTests.class);
+
+    @Test
+    public void testGettersAndSetters() throws Exception {
+        // See: https://github.com/codebox/javabean-tester
+        // The classes that should NOT be tested with the Bean are the
+        // ones that set and get List<> instances
+        JavaBeanTester.test(QuarterDept.class);
+    }
+
+    @Test
+    public void test_notEqualNull() throws Exception {
+        QuarterDept qd = new QuarterDept();
+        assertNotEquals(qd, null);
+    }
+
+    @Test
+    public void test_notEqualAnotherClass() throws Exception {
+        QuarterDept qd = new QuarterDept();
+        assertNotEquals(qd, new Object());
+    }
+
+    @Test
+    public void test_equalsSelf() throws Exception {
+        QuarterDept qd = new QuarterDept("20204","CMPSC",0);
+        assertEquals(qd, qd);
+    }
+
+    @Test
+    public void test_equalsAnother() throws Exception {
+        QuarterDept qd1 = new QuarterDept("20204","CMPSC",0);
+        QuarterDept qd2 = new QuarterDept("20204","CMPSC",0);
+        qd2.setDeptCode("CMPSC"); 
+        qd2.setQuarter("20204");
+        assertEquals(qd1, qd2);
+    }
```

Comment:
What is this testing? Wouldn't the setters do nothing here?

Reply:
That's left over from a refactoring... good catch. I'll remove them.

Reply:
Fixed